types/exif-js/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "private": true,
+    "name": "@types/exif-js",
+    "version": "2.3.9999",
+    "projects": [
+        "https://github.com/exif-js/exif-js#readme"
+    ],
+    "dependencies": {
+        "@types/node": "*"
+    },
+    "devDependencies": {
+        "@types/exif-js": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "Dmitry Zhukov",
+            "githubUsername": "darkoutx"
```

Comment:
Only add yourself, please

Reply:
Please remove this person, unless they comment here that they want to be an owner. We don’t want people signing others up for pings by @typescript-bot without their consent.

Reply:
I don't mind to be pinged and to update types in future if receive some new info about target package.
I just didn't want to reformat code after few months of silence past already since I've decided to make a PR.
P.S. Thanks, Artem, for mentioning me and helping with publishing

Reply:
@andrewbranch Dmitry has agreed to become a contributor to this package. See the comment above